src/workloads/query/TenMDocCollection_ObjectId_Sharded.yml

```diff
@@ -38,7 +38,6 @@ Actors:
           OperationCommand:
             shardCollection: *Namespace
             key: {_id: "hashed"}
-            numInitialChunks: 12
```

Comment:
Will older mongo versions still have a default of 2 chunks per shard and will that be significant?

Reply:
If the updated workload is ran against an older version, the default of 2 chunks per shard will created. It seems that using the default value should not affect the performance in this case.

Reply:
Can you confirm that with a quick patch?

Reply:
The performance of those workloads shall be the same, because data will still be evenly distributed across all shards, regardless of the number of chunks used.  

A patch using 6.0 is running [here](https://spruce.mongodb.com/version/6560a5632a60ed45b4d9ec1f/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).